salt_resources/states/mirrors/apt-mirror.list

```diff
@@ -0,0 +1,25 @@
+############# config ##################
+#
+# set base_path    /var/spool/apt-mirror
+#
+# set mirror_path  $base_path/mirror
+# set skel_path    $base_path/skel
+# set var_path     $base_path/var
+# set cleanscript $var_path/clean.sh
+# set defaultarch  <running host architecture>
+# set postmirror_script $var_path/postmirror.sh
+# set run_postmirror 0
+set nthreads     20
+set _tilde 0
+#
+############# end config ##############
+
+deb https://repo.saltstack.com/apt/debian/8/amd64/latest jessie main
```

Comment:
We can probably just update the package-sources for this and use that, right? Maybe prepend whatever the config is doing to that on the fly?